docker/elrond/Dockerfile

```diff
@@ -1,29 +1,18 @@
-FROM centos:7
+FROM golang:1.13.6 as builder
 MAINTAINER ElrondNetwork
-ARG CACHE_CANCELER=unknown
 
-# prepare
-RUN yum install -y wget git curl gcc golang && \
-    rm -rf /var/lib/apt/lists/*
+WORKDIR /go/elrond-go
+COPY . .
+RUN GO111MODULE=on go mod vendor
+#Elrond node
+WORKDIR /go/elrond-go/cmd/node
+RUN go build -i -v -ldflags="-X main.appVersion=$(git describe --tags --long --dirty)"
 
-# install go
-RUN wget https://storage.googleapis.com/golang/go1.11.1.linux-amd64.tar.gz && \
-    tar -C /usr/local -xzf go1.11.1.linux-amd64.tar.gz && \
-    rm go1.11.1.linux-amd64.tar.gz
-
-
-# configure go
-RUN mkdir -p /go/src /go/bin && chmod -R 777 /go
-ENV GOPATH /go
-ENV PATH /usr/local/go/bin:/go/bin:$PATH
-WORKDIR /go
-
-# fetch repository
-WORKDIR src/github.com/ElrondNetwork
-RUN ls
-
-COPY ./ ./elrond-go/
-WORKDIR elrond-go
-RUN >> skipP2PMessengerTests
-RUN ls
-RUN GO111MODULE=on go mod vendor
\ No newline at end of file
+# ===== SECOND STAGE ======
+FROM ubuntu
```

Comment:
Should we maybe lock ubuntu to 18.04 just to be on par with the other testnets ?

Reply:
Yup, this is my only remark too 👍

Reply:
sure, totally makes sense